content/02.main-text.md

```diff
@@ -144,6 +144,19 @@ We annotated the author list to indicate that author order was partly randomized
 
 ### Discussion
 
+The Deep Review was not the first scholarly manuscript written online via an open collaborative process.
+Interestingly, the collaborative scholarly works (of which we're aware) tend to have dealt with rapidly developing topics.
+In 2013, two dozen mathematicians created the 600-page "HoTT Book" on "Homotopy Type Theory", writing collaboratively in LaTeX on GitHub [@url:https://homotopytypetheory.org/book/; @url:http://math.andrej.com/2013/06/20/the-hott-book/].
+More recently, Andreas Antonopoulos has led the writing of two technical books on cryptocurrency --- titled [Mastering Bitcoin](https://github.com/bitcoinbook/bitcoinbook) and [Mastering Ethereum](https://github.com/ethereumbook/ethereumbook) --- on GitHub in asciidoc format.
+Both books have engaged hundreds of contributors.
+Mastering Ethereum has yet to be published, while Mastering Bitcoin continues to be maintained and expanded since its 2014 publication by O'Reilly Media.
+A 2017 perspective on the future of peer review was written collaboratively on Overleaf, with contributions from 32 authors [@doi:10.12688/f1000research.12037.2].
+Also in 2017, while debate was raging over tightening the default threshold for statistical significance, nearly 150 scientists contributed to a Google Doc, which reached 100-pages in length [@doi:10.1126/science.aat0471].
+The project lead then condensed this discussion into a more traditional journal commentary titled "Justify your alpha" with 88 authors [@doi:10.1038/s41562-018-0311-x].
+The greatest success to date of open collaborative writing is arguably Wikipedia, whose English version contains over 5.5 million articles.
+Wikipedia scaled encyclopedias [far beyond](https://en.wikipedia.org/wiki/Wikipedia:Size_comparisons) any privately-written alternative.
+As the examples above illustrate, open collaborative writing appears to scaling scholarly manuscripts where diverse opinion and expertise are paramount beyond what would otherwise be possible.
```

Comment:
Should `appears` be `appeals`?

Reply:
I meant "appears to scale" rather than "appeals to scaling" but would be fine with either.

Reply:
`appears to scale` sounds good